service/src/test/java/bio/terra/cbas/controllers/VerifyPactsAllControllers.java

```diff
@@ -123,6 +129,13 @@ public void initializeDAO() throws Exception {
     when(runDao.createRun(any())).thenReturn(1);
   }
 
+  @State({"cromwell initialized"})
```

Comment:
I think we should try to manage our states in a more modular way - this is the start of an attempt to do so